bcitflex/templates/courses/index.html

```diff
@@ -6,7 +6,7 @@ <h1>{% block title %}Courses{% endblock %}</h1>
 
 {% block content %}
     <div class="filter">
-      <form action="/courses" method="POST">
+      <form action="/courses" method="GET">
```

Comment:
Could you please share why the switch to GET? Forgive my ignorance here, so feel free to link me to a resource.

Reply:
@jonbiemond  GET is best practice for getting list of objects out of DB, using query and queryParams. Also for implementing requested pagination.